frontend/app/views/fragments/form/cardDetailPayPal.scala.html

```diff
@@ -0,0 +1,189 @@
+
+@import com.gu.salesforce.PaidTier
```

Comment:
This file duplicates cardDetail.scala.html but will be removed when we finally release PayPal